packages/fluent/scss/calendar/_layout.scss

```diff
@@ -109,6 +109,11 @@
         display: none;
     }
 
+    .k-calendar-td.k-empty {
+        pointer-events: none;
+        user-select: none;
```

Comment:
I think we don't need **user-select: none** here, as it is inherited from _.k-calendar_